src/compute/src/lib.rs

```diff
@@ -43,15 +43,18 @@ pub enum AsyncStackTraceOption {
 /// Command-line arguments for compute-node.
 #[derive(Parser, Clone, Debug)]
 pub struct ComputeNodeOpts {
-    // TODO: rename to listen_address and separate out the port.
-    #[clap(long, default_value = "127.0.0.1:5688")]
+    // TODO: rename to listen_addr and separate out the port.
+    /// The address that this service listens to.
+    /// Usually the localhost + desired port.
+    #[clap(long, alias = "listen-addr", default_value = "127.0.0.1:5688")]
     pub host: String,
```

Comment:
🤔 Shall we change it to something like the following: 
 ```suggestion
    #[clap(long, alias = "host", default_value = "127.0.0.1:5688")]
    pub listen_addr: String,
```
So we don't need to modify the related codes where the host is referenced when we deprecate it in the next release.

Reply:
oops, indeed, I forgot to change it back